### DISCUSSION TITLE
How helpful was this tool in your organisation and project integration?

### DISCUSSION THREAD

**Discussion by PaulGi48** [jgraph/drawio-desktop#1071] [General]
I had a fun time working with this tool and i wish i could actually appreciate the efforts of the creator in other ways .
Just want to say a big thank you to Gaudenz Alder and Jgraph team!!!

**Comment by davidjgraph** [comment]
We're planning a lifesize golden statue of Gaudenz. We'll post an address where people can send flowers and cheese...
